src/mbgl/renderer/layers/render_background_layer.cpp

```diff
@@ -118,6 +126,12 @@ void RenderBackgroundLayer::render(PaintParameters& parameters) {
             );
         }
     } else {
+        auto renderPass = (evaluated.get<BackgroundColor>().a >= 1.0f
```

Comment:
`allowedPass` ?

Reply:
-> backgroundRenderPass, to be consistent with fill implementation.